dyn_em/module_diffusion_em.F

```diff
@@ -5237,10 +5238,15 @@ FUNCTION pthnl(d,h)
    REAL,PARAMETER :: b1 = 2.0, b2 = 0.875
    real           :: d,h,doh,num,den
 
-   doh   = d/h
-   num   = a1*(doh)**b1 + a2*(doh)**b2+a3
-   den   = a4*(doh)**b1 + a5*(doh)**b2+a6
-   pthnl = a7*num/den + (1. - a7)
+   if (h .ne. 0) then
+      doh   = d/h
+      num   = a1*(doh)**b1 + a2*(doh)**b2+a3
+      den   = a4*(doh)**b1 + a5*(doh)**b2+a6
+      pthnl = a7*num/den + (1. - a7)
+   else
+      pthnl = 1.
+   endif
+
```

Comment:
this and the following 2 changes are as in PR https://github.com/wrf-model/WRF/pull/1219

Reply:
Very good!